### PR TITLE
Feature/calculator v3

### PR DESCRIPTION
**Pull request by andivido**
Adds version 3 of the widget to the plugin

### CODE REVIEW

File: upload/catalog/view/theme/default/template/extension/payment/financePlugin.twig [hunk 1]
```diff
@@ -32,7 +31,7 @@
 <script>
 (function($) {
     $(function () {
-
+        console.log($("#financePluginForm").serializeArray());
```
Comment (JonnyCarter) [inline upload/catalog/view/theme/default/template/extension/payment/financePlugin.twig]: Can you remove this

File: upload/catalog/controller/extension/module/financePlugin_calculator.php [hunk 2]
```diff
@@ -45,11 +54,13 @@ public function index() {
...
 
 		$data = array(
-			'api_key'					=> $js_key,
-			'product_price'				=> $localised_price,
-			'plan_list'					=> $plans_list,
-			'generic_credit_req_error'	=> 'Credit request could not be initiated',
-			'environment'	=> $this->config->get('payment_financePlugin_environment')
+			'api_key'				   => $js_key,
+			'product_price'		 	   => $localised_price*100,
+			'plan_list'				   => $plans_list,
```
Comment (JonnyCarter) [inline upload/catalog/controller/extension/module/financePlugin_calculator.php]: Can you fix formatting

File: upload/catalog/view/theme/default/template/extension/payment/financePlugin.twig (hunk 1 above)
Comment (andivido) [reply upload/catalog/view/theme/default/template/extension/payment/financePlugin.twig]: Removed

File: upload/catalog/controller/extension/module/financePlugin_calculator.php (hunk 2 above)
Comment (andivido) [reply upload/catalog/controller/extension/module/financePlugin_calculator.php]: Removed rogue tabs weirding out the formatting

**Pull request merged**